java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -47,4 +49,17 @@ protected static CommandManager buildCommandManager(ChannelHandler channelHandle
     protected RedisClient(ConnectionManager connectionManager, CommandManager commandManager) {
         super(connectionManager, commandManager);
     }
+
+    /**
+     * Executes a single custom command, without checking inputs. Every part of the command, including
+     * subcommands, should be added as a separate value in args.
+     *
+     * @param args command and arguments for the custom command call
+     * @return CompletableFuture with the response
+     */
+    public CompletableFuture<Object> customCommand(String[] args) {
```

Comment:
Yes.  Good point.  Let me double check with the cluster client and make sure that's correct .  
It's hard to see in this PR since it doesn't yet have the `RedisClusterClient`.

Reply:
Update:  the return typing differs between the `RedisClient` and the `RedisClusterClient`.  This is because of routing, and we won't be able to merge the calls until we have added the response typing (`RedisFuture`) that helps distinguish between a routed response and a single value response.